### DISCUSSION TITLE
how to set a default templat

### DISCUSSION THREAD

**Discussion by emacs25** [emacs-gnuplot/gnuplot#86] [Q&A]
for example,i want set these options by default when using run-gnuplot

set term svg size 800,600 samples 200 font "xxx,12"

how should i do with emacs lisp?

thanks everybody

**Comment by E3D3** [comment]
Maybe [https://github.com/hesstobi/Gnuplot-Templates](https://github.com/hesstobi/Gnuplot-Templates) is interesting for you.
